project/Rholang.scala

```diff
@@ -9,34 +9,6 @@ import sbt.io.Path.relativeTo
  */
 object Rholang {
```

Comment:
Can we delete the whole file? IntelliJ show no usage for `lazy val rholangSettings`.

Reply:
Nevermind, we use it in `build.sbt`. IntelliJ fooled me.

Reply:
But what about moving it to `build.sbt` and deleting this file?